lnd/neutrino/lnd.conf

```diff
@@ -37,13 +37,12 @@ listen=0.0.0.0:9735
 
 ; Specify the interfaces to listen on for gRPC connections.  One listen
 ; address per line.
-rpclisten=127.0.0.1:10009
-rpclisten=192.168.83.33:10009
+rpclisten=0.0.0.0:10009
```

Comment:
We sure this is a good idea?

Reply:
Its fine within a container or with a decent firewall :P

Reply:
Agree.  Let's just remember to `ufw`, or something if that config is to ever run outside of the container :)

Reply:
noma installs it all inside a container

Reply:
I know.  Just wanted to leave a trace.